tools/apollo/create_or_update_organism.py

```diff
@@ -25,6 +36,7 @@
     parser.add_argument('--public', action='store_true', help='Make organism public')
     parser.add_argument('--group', help='Give access to a user group')
     parser.add_argument('--remove_old_directory', action='store_true', help='Remove old directory')
+    parser.add_argument('--use_remote_user', type=str2bool, default=False, help='Authentification with remote_user')
```

Comment:
You can use `action='store_true'`here, will just act as a flag on the cli and return a bool True/False.

Reply:
For galaxy, with `action='store_true'`, the galaxy side would be something like

```
<command>
... $use_remote_user ...
</command>
<inputs>
  <param name="use_remote_user" type="boolean" truevalue="--use_remote_user" falsevalue="" ... />
</inputs>
```

which is pretty commonly used across IUC tools I feel.

Reply:
I agree, but it would leave the decision making to the tool user, who might not know if 'remote_user' is used. (The only thing it changes is the history step with the credentials, so it's not really an issue, but it might be confusing)

Reply:
Ahhh I see what you're saying, galaxy would provide that variable automatically from the configuration file or something. This makes way more sense as to why you used the approach you did.